Announce AWS marketplace offering (#1049)
flowforge/website: src/blog/2023/08/aws-marketplace-announce.md

---
title: FlowFuse is now available on AWS Marketplace
subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
date: 2023-08-18
authors: ["ian-skerrett"]
image: blog/2023/08/images/UNS/UNS-Headline.svg
tags:
    - posts
    - news

> ---

Reviewer: ```suggestion
---

```

---
title: FlowFuse is now available on AWS Marketplace
subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.
date: 2023-08-18
authors: ["ian-skerrett"]

> image: blog/2023/08/images/UNS/UNS-Headline.svg

Reviewer: Consider an image from unsplash if you want to publish today: https://unsplash.com/photos/M2524ncJQ40

tags:
    - posts
    - news
---

> Today, we are excited to announce that FlowFuse is now available from the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). This makes it very easy for customers to install and run Node-RED and FlowFuse on the AWS Cloud.

Reviewer: Nit, but I like this a bit better

```suggestion
Many customers want to run FlowFuse in their own cloud environment, AWS being a great example. Today we're excited to announce that FlowFuse is now available from the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). This makes it very easy for customers to install and run Node-RED and FlowFuse within minutes.

```

- Team collaboration for Node-RED developers, allowing multiple developers to work together on a single Node-RED instance, including the ability to have an audit log of changes.
- DevOps deliver pipelines that support a software development lifecycle for Node-RED development. Pipelines can be setup to establish development, test and production environments for Node-RED instances.
- Snapshot of Node-RED instances to create a version history of changes to Node-RED applications. This also includes the ability to rollback to a previous version of a Node-RED instance.

FlowFuse on the AWS Marketplace is available under the Apache License c2.0 open source license. Customers can use FlowFuse free of charge but will need to pay AWS EC2 usage for hosting FlowFuse. FlowFuse offers commercially licensed Premimum and Enterprise tiers that includes enterprise oriented features, including:
- 24/7 technical support
- Single Sign-on
- High Availability for Node-RED applications
- Remote device management

> Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse.

Reviewer: ```suggestion
Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse. To understand what FlowFuse can do for your use-case, please [book a demo](/book-demo).
```

> FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](https://flowfuse.com/docs/install/docker/aws-marketpace/) and give it a try and let us know what you think.

Reviewer: ```suggestion
FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketpace/) and give it a try and let us know what you think.
```

---
title: FlowFuse is now available on AWS Marketplace
subtitle: Making is easier to run Node-RED and FlowFuse on AWS Cloud
description: This is an announcement that customers can install FlowFuse on AWS Cloud from the AWS Marketplace.

> date: 2023-08-18

Reviewer: ```suggestion
date: 2023-08-21
```

authors: ["ian-skerrett"]
image: blog/2023/08/images/flowfuse-aws.png
tags:
    - posts

>     - news

Reviewer: ```suggestion
    - news
    - flowfuse
```

- DevOps deliver pipelines that support a software development lifecycle for Node-RED development. Pipelines can be setup to establish development, test and production environments for Node-RED instances.
- Snapshot of Node-RED instances to create a version history of changes to Node-RED applications. This also includes the ability to rollback to a previous version of a Node-RED instance.

FlowFuse on the AWS Marketplace is available under the Apache License c2.0 open source license. Customers can use FlowFuse free of charge but will need to pay AWS EC2 usage for hosting FlowFuse. FlowFuse offers commercially licensed Premimum and Enterprise tiers that includes enterprise oriented features, including:
- 24/7 technical support
- Single Sign-on
- High Availability for Node-RED applications
- Remote device management

Customers can easily upgrade to the premium or enterprise tier by obtaining a commercial license from FlowFuse. To understand what FlowFuse can do for your use-case, please [book a demo](/book-demo).

> FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketpace/) and give it a try and let us know what you think.

Reviewer 2: ```suggestion
FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketplace/) and give it a try and let us know what you think.
```

> FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketplace/) and give it a try and let us know what you think.

Reviewer: ```suggestion
FlowFuse on AWS Marketplace is available [immediately](https://aws.amazon.com/marketplace/pp/prodview-3ycrknfg67rug?sr=0-1&ref_=beagle&applicationId=AWSMPContessa). Please refer to our [documentation](/docs/install/docker/aws-marketplace/#installing-flowfuse-from-aws-market-place) and give it a try and let us know what you think.
```
